packages/happo-target-react-native/src/defaultOptions.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+
+const iosAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/ios/build/Build/Products/Debug-iphonesimulator/HappoRunner.app',
+);
+
+const androidAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
```

Comment:
Do you need/want the trailing slash here when using `path.join`?

Also, I worry a little about using __dirname because this may unexpectedly break if this file moves. Probably okay if we have test coverage for this, but if we can find a less brittle way to express this, that might be a good idea.